documentation/src/docs/rest/restapis.md

```diff
@@ -88,7 +88,6 @@ Create a new scope
 
 ##### Request body
 ```json
-json :
```

Comment:
why are we removing the json element?

Reply:
This is an auto-generated document from the swagger yaml file. This does not cause any changes to formatting. The complete document can be seen @ https://github.com/shrids/pravega/blob/issue-fixRESTdocumentation/documentation/src/docs/rest/restapis.md

Reply:
So why are we checking in a generated document?